src/client/components/pages/Courses/MeetingModal.tsx

```diff
@@ -37,37 +37,61 @@ interface MeetingModalProps {
  * Utility component to style content within meeting modal body
  */
 const MeetingModalBodyGrid = styled.div`
-  display: grid;
-  grid-template-columns: repeat(2, 1fr);
-  grid-template-rows: repeat(2, 1fr);
-  grid-template-areas:
-  "meetingScheduler roomAvailability"
-  "facultyNotes roomAvailability";
+  height: 75vh;
+  width: 75vw;
+  display: flex;
+  flex-direction: row;
   column-gap: ${fromTheme('ws', 'xsmall')};
```

Comment:
This can be removed, since we're not using `grid` anymore

Reply:
Thanks for pointing this out. This was removed in c001d11.